tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -355,4 +355,79 @@ public function theFollowingMimeTypesShouldExistForUser(string $shouldOrNot, Tab
 			}
 		}
 	}
+
+	/**
+	 * @Then the app list response should contain the following information:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theAppListResponseShouldContainTheFollowingInformation(TableNode $table): void {
+		$responseArray = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse());
+
+		if (!isset($responseArray['mime-types'])) {
+			throw new Exception(__METHOD__ . "The response does not contain a 'mime-types' key.");
```

Comment:
use assert functions
```suggestion
			Assert::
```